src/lib/containers/download_list_v2/MeetAccessRequirementCard.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from 'react'
+import { useErrorHandler } from 'react-error-boundary'
+import { toError } from '../../utils/ErrorUtils'
+import { SynapseSpinner } from '../LoadingScreen'
+import useGetAccessRequirement from '../../utils/hooks/SynapseAPI/useGetAccessRequirement'
+import { AccessRequirement, SelfSignAccessRequirement } from '../../utils/synapseTypes'
+import { Button } from 'react-bootstrap'
+import { Icon } from '../row_renderers/utils'
+import { EASY_DIFFICULTY, MEDIUM_DIFFICULTY, VARIABLE_DIFFICULTY } from '../../utils/SynapseConstants'
+import AccessRequirementList, { SUPPORTED_ACCESS_REQUIREMENTS } from '../access_requirement_list/AccessRequirementList'
+
+export type MeetAccessRequirementCardProps = {
+  accessRequirementId: number
+  count: number
+}
+export const TERMS_OF_USE_TITLE = 'Requires Acceptance of Data-Specific Terms of Use'
+export const SELF_SIGN_TITLE = 'Requires Acceptance of Data-Specific Terms of Use'
+export const ACT_TITLE = 'Requires Approval of Data-Specific Access Requirements'
+export const LOCK_TITLE = 'Access Restricted'
```

Comment:
Pulled out the AR titles so that I could easily verify these were on the screen when expected (in the unit tests).